tests/accidental_tests.ts

```diff
@@ -281,6 +282,14 @@ function basic(options: TestOptions): void {
 
   f.draw();
 
+  // Render bounding boxes
+  if (options.params.bbox === true) {
```

Comment:
Minor suggestion:

bbox could be `drawBoundingBox`. It doesn't matter if we are more verbose since these are just tests....

Then the code can be:

```
 if (options.params.drawBoundingBox) {
```

Reply:
done